examples/simple/lint/L-STDLIBE2-good.c

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main()
+{
+    size_t array_size = 20;
```

Comment:
Does this need to be size_t instead of int?  One would expect the -good programs to only fix the -bad ones minimally.